web/src/main/java/co/mobileaction/example/web/exception/BaseException.java

```diff
@@ -0,0 +1,24 @@
+package co.mobileaction.example.web.exception;
+
+/**
+ * @author elif
+ * @date 05.07.2023
+ * @time 15.58
+ */
+public class BaseException extends RuntimeException
```

Comment:
```suggestion
@Getter
@Setter
public class BaseException extends RuntimeException
```
You can use getter setter like that. No need extra getter setter definition (if there is no specific implementation)